config/environment.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (environment) {
     },
 
     // defaults for running travis-web
-    apiEndpoint: 'https://api-staging.travis-ci.org',
+    apiEndpoint: 'https://api.travis-ci.org',
```

Comment:
I wish I had a better way to accomplish this, but if your branch has `staging` in the name, you get `org-staging` and `com-staging` PR deployments too, which removes the need for this sort of thing.

Reply:
This was actually only for testing locally, I just forgot to remove it. But thanks for the tip!